### PR TITLE
 #FN-144 chore: create top navigation bar and footer

### PR DESCRIPTION
**Pull request by EvelyneUmubyeyi**
# PR Description
In this PR I created a top nav bar and footer to be used in the web app.

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] create the top navigation bar of the web app
- [x] create the footer of the web app

## Description of tasks that were expected to be completed and that are not yet completed

- [x] None

## How this may be tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `yarn i` to install all required dependencies
2. run `yarn run dev`

## Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case it's appropriate and applicable

## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: src/components/Nav/navBar.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        <img src={Logo} alt="logo" className="lg:h-9 lg:w-36 md:w-20 md:h-6 md:mr-2 custom-md: w-[6rem] custom-md: h-[1.5rem]"></img>
+        <div className="flex">
+          <div className="flex items-center mr-4 xl:mr-10 md:mr-2">
+            <p className="pr-1 custom-md: text-sm md:text-base">Category</p>
+            <FontAwesomeIcon className="text-sm" icon={faChevronDown} />
+          </div>
+          <p className="mr-4 xl:mr-10 md:mr-2 custom-md: text-sm md:text-base">What’s New</p>
+          <p className="custom-md: text-sm md:text-base custom-md:mr-0">Contact Us</p>
+        </div>
+        <div className="border-[1px] border-[#9C9EBA] text-[#9C9EBA] xl:w-[40%] md:w-[30%] py-1.5 px-3 lg:px-4 md:mr-2 rounded-3xl flex items-center justify-between custom-md:w-[30%]">
```
Comment (dushimeemma) [inline src/components/Nav/navBar.tsx]: I am wondering the reason why this is not a search field!?

**Review by dushimeemma — CHANGES_REQUESTED** [review]
Would you mind safely deleting the Greet component and Greet test files?

**Review by Charlesntwari — APPROVED** [review]

File: .storybook/main.ts
Comment (Honore123) [inline .storybook/main.ts:1]: Delete comments

**Review by dushimeemma — APPROVED** [review]
Good job 🥇

**Review by Honore123 — APPROVED** [review]

**Pull request merged**